int8 device # integer corresponding to X in /dev/videoX
int8 resolution # 0 means turn off, 1-2-3-4-5 means low to high res